symbols/arm9.yml

```diff
@@ -6997,6 +7031,16 @@ arm9:
         Gets the special episode type from the EXECUTE_SPECIAL_EPISODE_TYPE script variable.
         
         return: special episode type
+    - name: IsSpecialEpisodeOpen
+      address:
+        EU: 0x204CC84
+        NA: 0x204C94C
+        JP: 0x204CCAC
+      description: |-
+        Checks if a special episode is unlocked from the SPECIAL_EPISODE_OPEN script variable.
+        
+        r0: index
```

Comment:
Do you know what the index here means? Is it the same as the special episode type?

Reply:
Oops yeah, my bad